packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -204,7 +204,8 @@ function useMarginProps( name ) {
 function useBlockGapProps( name ) {
 	const [ gapValue, setGapValue ] = useStyle( 'spacing.blockGap', name );
 	const resetGapValue = () => setGapValue( undefined );
-	const hasGapValue = () => !! gapValue;
+	const [ userSetGapValue ] = useStyle( 'spacing.blockGap', name, 'user' );
+	const hasGapValue = () => !! userSetGapValue;
```

Comment:
If this winds up working okay, we might want to roll it out for padding and margin, too 🤔

Reply:
I've added it to Padding since that is exposed at the root level, too, which appears to be where we ran into the main issue.